ui/app/common/displaycontrols/observation/services/formRecordTreeBuildService.js

```diff
@@ -171,6 +167,20 @@ angular.module('bahmni.common.displaycontrol.observation')
             });
         };
 
+        self.setTranslatedShortName = function (translationData, translationKey, obsGroup, defaultShortName) {
+            if (self.isTranslationKeyPresent) {
```

Comment:
Hi @vinishad, ```isTranslationKeyPresent``` is a method. I think you have to call the method to get the result, isn't it?

Reply:
yes mybad will update it